sdk/core/azure-core/src/main/java/com/azure/core/util/paging/ContinuablePage.java

```diff
@@ -0,0 +1,20 @@
+package com.azure.core.util.paging;
+
+import java.util.List;
+
+/**
+ * Represents Page from service that has reference to next set of one or more pages.
+ *
+ * @param <C> Type of the continuation token
+ * @param <T> Type of the items in the page
+ */
+public interface ContinuablePage<C, T> {
+    /**
+     * @return list of items in the page.
+     */
+    List<T> getItems();
```

Comment:
We had a discussion with Cosmos about making this be an Iterable or IterableStream. Did you have any reason to keep it as a List?

Reply:
The only reason I didn't replace `List` with  Iterable/IterableStream was to avoid breaking change. Should we add a getter that returns IterableStream and mark `List<T> getItems()` deprecated? how about the name `getElements()` for the new getter? thoughts?

Reply:
PR is updated by adding `ContinuablePage::getElements()`. In `Page` (that extends from `ContinuablePage`) the `getItems()` method is marked as deprecated with default implementation based `getElements()`.